src/range_proof/mod.rs

```diff
@@ -1,13 +1,10 @@
 #![allow(non_snake_case)]
-#![cfg_attr(feature = "docs", doc(include = "../../docs/range-proof-protocol.md"))]
+#![doc = include_str!("../../docs/range-proof-protocol.md")]
 
 extern crate alloc;
-#[cfg(feature = "std")]
-extern crate rand;
 
-#[cfg(feature = "std")]
-use self::rand::thread_rng;
 use alloc::vec::Vec;
+use rand::thread_rng;
```

Comment:
Done. (I think I did this to make curve25519-dalek compile because it no longer had the std feature or something... just noting this here because I'll have to make some sort of change to curve25519-dalek's usage of the std feature when the time comes).